src/net/sf/memoranda/ui/App.java

```diff
@@ -98,6 +107,53 @@ else if (
 		}
 		if (!Configuration.get("SHOW_SPLASH").equals("no"))
 			splash.dispose();
+		
+		// ajcallos 2/16/2016
+		if (SystemTray.isSupported()){
+			try {
+				initTray();
+			} catch (AWTException ex){
+				//TODO:
+			}
+		} 
+	}
+	
+	/** 
+	 * Initializes the Memoranda icon in the system tray
+	 * Uses the oracle example at https://docs.oracle.com/javase/tutorial/uiswing/misc/systemtray.html as the baseline
+	 * @author ajcallos 
+	 * added 2/16/2016 
+	 */
+	private void initTray() throws AWTException	{
+		final String iconPath = "resources/memorandaTrayIcon.gif";
+		final PopupMenu trayPopupMenu = new PopupMenu();
+		final TrayIcon trayIcon = new TrayIcon(new ImageIcon(App.class.getResource(iconPath)).getImage());
+		final SystemTray tray = SystemTray.getSystemTray();
+		
+		MenuItem exitItem = new MenuItem("Exit");
+		
+		trayPopupMenu.add(exitItem);
+		
+		trayIcon.setPopupMenu(trayPopupMenu);
+		tray.add(trayIcon);
+		
+		// Sets exit action
+		exitItem.addActionListener(new ActionListener(){
+			public void actionPerformed(ActionEvent e) {
+				tray.remove(trayIcon);
+				System.exit(0);
```

Comment:
I noticed that both doExit() and doMinimize() in AppFrame.java call AppFrame.exitNotify() before calling System.exit(), perhaps it would be best to call that here before calling System.exit()?